webapp/channels/src/utils/paste.tsx

```diff
@@ -140,3 +146,40 @@ export function formatMarkdownLinkMessage({message, clipboardData, selectionStar
     const markdownLink = `[${selectedText}](${clipboardUrl})`;
     return markdownLink;
 }
+
+export function pasteHandler(event: ClipboardEvent, location: string, message: string, isNonFormattedPaste?: boolean, caretPosition?: number) {
```

Comment:
This function is only used in the advanced text editor, whats the reason for this function to be in commons?

Reply:
[Functional fixedness](https://en.wikipedia.org/wiki/Functional_fixedness). I moved it on my first approach, and now I have left it there.

It may be useful if we want to add this paste handler to other places, but if we want to do that, we can move it later.

Also, feels more correct to test it here than in the component tests, but I can figure out the best way to do it clear.

Reply:
I just saw that we should use it also in `edit_post.tsx` but it is not so straightforward. So... will keep it in utils/paste until I take the time to deal with edit_post.tsx.